commands/roll.js

```diff
@@ -17,66 +17,10 @@ const DICE_RANGE_ICONS = {
 
 module.exports = {
 	name: 'roll',
-	aliases: ['r', 'lance', 'lancer', 'slå', 'sla'],
+	aliases: ['r', 'lance', 'lancer', 'slå', 'sla', 'würfeln', 'wurf'],
```

Comment:
I think it would be best to remove any translation in the command aliases array (french and swedish too) or it'll become crowded after just a few supported languages.

Reply:
Yeah, i wanted to make it easier for my players, but i see your point.

Reply:
I pushed a new commit where i removed the language aliases from commands, other than the english language one